bedrock/firefox/templates/firefox/features/index.html

```diff
@@ -113,7 +113,7 @@ <h3 class="mzp-c-card-picto-title">{{ ftl('features-shared-customize-your-browse
           <h3 class="mzp-c-card-picto-title">{{ ftl('features-shared-sync-between-devices') }}</h3>
           <div class="mzp-c-card-picto-desc">
             <p>{{ ftl('features-index-important-stuff') }}</p>
-            <a class="mzp-c-cta-link" href="{{ url('firefox.accounts') }}">{{ ftl('features-index-get-an-account') }}</a>
+            <a class="mzp-c-cta-link" href="{{ url('firefox.sync') }}">{{ ftl('features-index-get-an-account') }}</a>
```

Comment:
Should we also change the text to "Learn more" instead of "Get a Firefox Account"? We have the generic string: `{{ ftl('ui-learn-more') }}`

Reply:
I thought this text was still appropriate because they do need to click the "sign up" button on the sync page.

Death to "learn more" 😛

Reply:
Fair point. I too hate "learn more" as a CTA.